### PR TITLE
K5 font change

### PR DESCRIPTION
**Pull request by vargaat**
refs: MBL-15371
affects: Student
release note: none

test plan: K5 mode should use the balsamiq font across the application.

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/79427330/p/6d8806b84111a0f50baaab4df6467e72">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/79427341/p/faa33bafc76be46a22896d6639517c1d">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/79427287/p/1db3f74669c51064e70488ad9f54418e">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.79% | 91.92% | -0.12%
Core/Core/Dashboard/K5/ViewModel/Grades/K5GradesViewModel.swift | 0% | 0% | 0%
Core/Core/Dashboard/K5/ViewModel/Schedule/K5ScheduleViewModel.swift | 47.37% | 47.37% | 0%
Core/Core/Dashboard/K5/ViewModel/Resources/K5ResourcesViewModel.swift | 0% | 0% | 0%
Core/Core/Extensions/FontExtensions.swift | 25% | -- | --

[MBL-15371](https://instructure.atlassian.net/browse/MBL-15371)

#### Affected Apps: Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Extensions/UINavigationBarExtensions.swift [hunk 1]
```diff
@@ -60,5 +62,12 @@ extension UINavigationBar {
         barTintColor = .backgroundLightest
         barStyle = .default
         isTranslucent = false
+        updateFontAppearance()
+    }
+
+    public func updateFontAppearance(useK5Fonts: Bool = AppEnvironment.shared.isK5Enabled) {
+        if useK5Fonts {
```
Comment (vargaat) [inline Core/Core/Extensions/UINavigationBarExtensions.swift:69]: This is unnecessary.

**Review by davidvarga93 — DISMISSED** [review]
QA+1

**Review by davidvarga93 — APPROVED** [review]

**Review by idevainstr — APPROVED** [review]
QA+1

**Pull request merged**